cmd/decrypt-all.go

```diff
@@ -4,12 +4,17 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"os"
+	"strconv"
 	"sync"
+	"time"
 
-	"github.com/agilebits/sm/secrets"
+	"github.com/josegonzalez/sm/secrets"
```

Comment:
Please revert all the `josegonzalez` references. They are there on purpose, on the off-chance they actually merge this in.

Reply:
i can't, because i made changes to those files now :)

Reply:
also been open for 1y now, so chances are likely 0 :)